tests/integration_tests/test_qwen_embedding_function.py

```
@@ -0,0 +1,215 @@
+            {"source": "vector_db"}
+        ]
+
+        collection.add(
+            ids=test_ids,
+            documents=test_documents,
+            metadatas=test_metadatas
+        )
+        print(f"   Added {len(test_documents)} documents")
+
+        # Query
+        query_text = "什么是嵌入？"
```

Ruff (0.14.10)

String contains ambiguous `？` (FULLWIDTH QUESTION MARK). Did you mean `?` (QUESTION MARK)?

(RUF001)